packages/open-truss/nextjs/src/app/ot/playground/[slug]/page.tsx

```diff
@@ -0,0 +1,36 @@
+import { promises as fs } from 'fs'
+import { parseYaml, applyConfiguration } from '@open-truss/open-truss'
+import * as COMPONENTS from '@/open-truss/components'
+import path from 'path'
+
+const configurationFunction = applyConfiguration(COMPONENTS)
+
+interface PlaygroundPage {
+  params: {
+    slug: string
+  }
+}
+
+export default async function Page({
+  params: { slug },
+}: PlaygroundPage): Promise<JSX.Element> {
+  let config
+  try {
+    const sanitizedSlug = path.basename(slug)
+    config = await fs.readFile(
+      `./src/open-truss/configs/${sanitizedSlug}.yaml`,
+      'utf-8',
+    )
+  } catch (err) {
+    if ((err as NodeJS.ErrnoException).code === 'ENOENT') {
+      return <>404 - File Not Found</>
```

Comment:
If we're going to 404 here we should probably legitimately 404. I'll add a commit that uses these instructions https://nextjs.org/docs/pages/building-your-application/routing/custom-error#reusing-the-built-in-error-page

Reply:
Those ended up being instructions for client side errors. I had to dig a bit to find the actual answer for rsc: https://github.com/vercel/next.js/discussions/53225

Fixed in 90280cff5e6aedc95a27dea6a9e0714b19acdedd